### PR TITLE
Feat/ex

### PR DESCRIPTION
**Pull request by pedrobertao**

### CODE REVIEW

File: src/scenes/Exchange/Sell/index.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    const estimatedCost = estimatedSellCost(firstTokenBalance, secondTokenBalance, sellAmount, secondTokenId === 'TRX')
+
+    if (estimatedCost <= 0) {
+      Alert.alert(tl.t('warning'), `Can\t trade ${estimatedCost} ${secondTokenId}`)
+      this.sellAmount.focus()
+      return
+    }
+
+    const { balance: userFirstTokenBalance } =
+      balances[publicKey].find(bl => bl.name === firstTokenId) || { balance: 0 }
```
[the remark this answers is not in the document]
Comment (pedrobertao) [reply src/scenes/Exchange/Sell/index.js]: I like this idea but this function needs to be broadly used to be considered to put to context

[the remark this answers is not in the document]
Comment (calebesantos) [reply src/scenes/Exchange/Sell/index.js]: Yeah, I agree. What I mean is just put in some shared class that has access to it or just create a shared function that the two components can use the function.

[the remark this answers is not in the document]
Comment (pedrobertao) [reply src/scenes/Exchange/Sell/index.js]: Done !

File: src/scenes/Exchange/ExchangeButton/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+import ButtonGradient from '../../../components/ButtonGradient'
+import * as Utils from '../../../components/Utils'
+import { Colors } from '../../../components/DesignSystem'
+import FadeIn from '../../../components/Animations/FadeIn'
+
+const ExchangeButton = ({text, loading, result, onSubmit}) => {
+  let element
+
+  if (loading) element = <ActivityIndicator color={Colors.primaryText} />
```
[the remark this answers is not in the document]
Comment (pedrobertao) [reply src/scenes/Exchange/ExchangeButton/index.js]: Done !

File: src/scenes/Exchange/ExchangeBalancePair/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import { formatNumber } from '../../../utils/numberUtils'
+
+const ARROW_GRADIENT = require('../../../assets/arrow-right-gradient.png')
+
+class ExchangeBalancePair extends Component {
+  render () {
+    const { balances, publicKey } = this.props.context
+    const { firstToken, secondToken, firstTokenImage, secondTokenImage } = this.props
+    const {balance: firstTokenBalance} = balances[publicKey].find(bl => bl.name === firstToken) || { balance: 0 }
+    const {balance: secondTokenBalance} = balances[publicKey].find(bl => bl.name === secondToken) || { balance: 0 }
```
[the remark this answers is not in the document]
Comment (pedrobertao) [reply src/scenes/Exchange/ExchangeBalancePair/index.js]: Done !

File: src/scenes/Exchange/ExchangeBalancePair/index.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+    return <Utils.View marginX={Spacing.large} paddingY='small'>
```
[the remark this answers is not in the document]
Comment (pedrobertao) [reply src/scenes/Exchange/ExchangeBalancePair/index.js]: Done !

**Pull request merged**